nifi-nar-bundles/nifi-standard-services/nifi-mongodb-services-bundle/nifi-mongodb-services/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Does this file need to be here? I know there's one in nifi-lookup-services but that might've been an oversight.